homeassistant/components/hunterdouglas_powerview/sensor.py

```diff
@@ -91,3 +91,7 @@ def _async_update_shade_from_group(self):
         """Update with new data from the coordinator."""
         self._shade.raw_data = self.data.get_raw_data(self._shade.id)
         self.async_write_ha_state()
+
```

Comment:
We shouldn't be creating these entities if its set to hard wired but thats not for this PR

Reply:
Ok - so we will just display battery entities for hardwired at this stage yea ? 
My hardwired currently are always at 90% anyway so no end user change there

The change comes when we remove the entity as we need to remove the old sensor.

Reply:
Right now no changes here besides moving async_update which was needed for the other change.

In the future we can simplify avoid creating the entities if it is hard wired and advise users to delete them (we could do it for them as well if we want to build that).